stable/distribution/UPGRADE_NOTES.md

```diff
@@ -0,0 +1,64 @@
+# JFrog distribution Chart Upgrade Notes
+This file describes special upgrade notes needed at specific versions
+
+## Upgrade from 3.x to 4.X (Chart Versions)
+
+* To upgrade from a version prior to 3.x, you first need to upgrade to latest version of 3.x as described in https://github.com/jfrog/charts/blob/master/stable/distribution/CHANGELOG.md.
+
+**DOWNTIME IS REQUIRED FOR AN UPGRADE!**
+* PostgreSQL sub chart was upgraded to version `8.x.x`. This version is not backward compatible with the old version (`0.9.5`)!
+* Note the following **PostgreSQL** Helm chart changes
+  * The chart configuration has changed! See [values.yaml](values.yaml) for the new keys used
+  * **PostgreSQL** is deployed as a StatefulSet
+  * See [PostgreSQL helm chart](https://hub.helm.sh/charts/stable/postgresql) for all available configurations
+* Upgrade
+  * Due to breaking changes in the **PostgreSQL** Helm chart, a migration of the database is needed from the old to the new database
+  * The recommended migration process Full DB export and import of Postgresql
```

Comment:
Missing "is".
i.e. `The recommended migration process is Full DB export and import of Postgresql`